misc/acrn-config/launch_config/com.py

```diff
@@ -441,46 +442,48 @@ def dm_arg_set(names, sel, dm, vmid, config):
     if is_nuc_clr(names, vmid):
         print("   --pm_notify_channel uart \\", file=config)
         print('   --pm_by_vuart pty,/run/acrn/life_mngr_$vm_name  \\', file=config)
-        print('   -s 1:0,lpc -l com2,/run/acrn/life_mngr_$vm_name \\', file=config)
+        print('   -l com2,/run/acrn/life_mngr_$vm_name \\', file=config)
 
     # mac_seed
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS"):
         print("   --mac_seed $mac_seed \\", file=config)
 
     # hard rt os
     if uos_type == "PREEMPT-RT LINUX":
-        print("acrn-dm -A -m $mem_size -s 0:0,hostbridge -U {} \\".format(scenario_uuid[vmid]), file=config)
+        print("acrn-dm -A -m $mem_size -s 0:0,hostbridge -U {} \\".format(scenario_uuid[vmid + sos_vmid]), file=config)
         #print("   -k /usr/lib/kernel/default-iot-lts2018-preempt-rt \\", file=config)
         print("   -k /usr/lib/kernel/default-iot-lts2018-preempt-rt \\", file=config)
         print("   --lapic_pt \\", file=config)
         print("   --rtvm \\", file=config)
-        print("   -s {},virtio-net,tap0 \\".format(launch_cfg_lib.virtual_dev_slot("virtio-net")), file=config)
         print("   --virtio_poll 1000000 \\", file=config)
         print("   --pm_notify_channel uart --pm_by_vuart tty,/dev/ttyS1 \\", file=config)
+        if not sel.bdf['ethernet'][vmid]:
```

Comment:
Please check this vmid need to translation to vmid+sos_vmid

Reply:
only need for uuid translation, not need for pt device